Kickstarter-iOS/ViewModels/FacebookLoginViewModelTests.swift

```diff
@@ -0,0 +1,31 @@
+@testable import Kickstarter_Framework
+@testable import Library
+import ReactiveExtensions_TestHelpers
+import ReactiveSwift
+import XCTest
+
+final class FacebookLoginViewModelTests: TestCase {
```

Comment:
The only test that was testing this stuff copied from `AppDelegateViewModelTests`.